src/styles/blocks/watch.scss

```diff
@@ -0,0 +1,74 @@
+.watch {

```

Comment:
You can add to hands one general class like - "watch_ hand" and put all of the repeated code there.  Here a lot of repeated code.  Hand positioning is almost the same.

Reply:
There i have diffrent properties for hands, like width and height, so positions for theese element are diffrent and related with it size. Only transform-origin the same for this element.